pkg/bpsi/receiver.go

```diff
@@ -36,12 +36,12 @@ func (r *Receiver) Intersect(ctx context.Context, n int64, identifiers <-chan []
 
 	// stage 1: read the bloomfilter from the remote side
 	stage1 := func() error {
-		if _bf, _, err := ReadFrom(r.rw); err == nil {
-			bf = _bf
-			return nil
-		} else {
+		_bf, _, err := ReadFrom(r.rw)
```

Comment:
You assign the output of `ReadFrom` to `_bf` and then later reassign to `bf`. From what I can see in the docs, a leading underscore has no convention. Can we just assign directly to `bf`?